core/src/main/java/hudson/model/ParametersAction.java

```diff
@@ -272,7 +284,12 @@ public ParametersAction merge(@CheckForNull ParametersAction overrides) {
         return parametersAction;
     }
 
+    @SuppressWarnings("unused")
     private Object readResolve() {
+        wasInitializedWithNullParameters = parameters == null;
```

Comment:
Well, he can find the root cause (e.g. Script user). Or create a bug. The logging is FINE now